conans/client/conf/detect.py

```diff
@@ -140,18 +140,28 @@ def _detect_compiler_version(result, output):
         elif compiler == "gcc":
             result.append(("compiler.libcxx", "libstdc++"))
             if Version(version) >= Version("5.1"):
-
+                # profile name is only the profile name
+                if profile_name == os.path.basename(profile_name):
```

Comment:
Overly complicated. Please pass the full path to the profile as argument, it is always available

Reply:
Done! :+1: